src/components/KMultiselect/KMultiselectItem.vue

```diff
@@ -41,6 +60,11 @@ const props = defineProps({
     // Items must have a label and value
     validator: (item: Record<string, string | number | boolean>): boolean => item.label !== undefined && item.value !== undefined,
   },
+  disabledTooltipText: {
+    type: String,
+    // Default tooltip text
+    default: 'This item is readonly',
+  },
```

Comment:
question: why are we adding this new prop? I thought we were just adding the icon?

Reply:
Fixed.